applications/enums.py

```diff
@@ -11,3 +11,10 @@ class Labels:
         ON_TRESTLES = _("On trestles")
         ON_TRAILER = _("On a trailer")
         UNDER_TARP = _("Under a tarp")
+
+
+class ApplicationStatus(Enum):
+    PENDING = "pending"
+    OFFER_GENERATED = "offer_generated"
+    OFFER_SENT = "offer_sent"
+    HANDLED = "handled"
```

Comment:
please, add `Labels`

Reply:
Isn't that gonna be handled by the front-end?

Reply:
these `Labels` are for the django-admin 🙂

Reply:
Aha! Got it. :)